libs/beacon/include/beacon/entropy.hpp

```diff
@@ -34,7 +34,7 @@ struct Entropy
 
   uint64_t       round{0};
   ConstByteArray seed{"genesis"};
-  ConstByteArray entropy{};
+  ConstByteArray entropy{"====DEFAULT====="};
```

Comment:
@troelsfr there was a bug where requesting the entropy too early caused a segfault by reinterpret_cast-ing this entropy